pkg/controller/pxcbackup/controller.go

```diff
@@ -500,20 +631,25 @@ func (r *ReconcilePerconaXtraDBClusterBackup) updateJobStatus(bcp *api.PerconaXt
 
 	bcp.Status = status
 
-	if status.State == api.BackupSucceeded {
+	switch status.State {
+	case api.BackupSucceeded:
+		log.Info("Backup succeeded")
+
 		if cluster.PITREnabled() {
-			collectorPod, err := binlogcollector.GetPod(context.TODO(), r.client, cluster)
+			collectorPod, err := binlogcollector.GetPod(ctx, r.client, cluster)
 			if err != nil {
 				return errors.Wrap(err, "get binlog collector pod")
 			}
 
-			if err := binlogcollector.RemoveGapFile(context.TODO(), cluster, r.clientcmd, collectorPod); err != nil {
+			log.V(1).Info("Removing binlog gap file from binlog collector", "pod", collectorPod.Name)
+			if err := binlogcollector.RemoveGapFile(ctx, cluster, r.clientcmd, collectorPod); err != nil {
```

Comment:
Not related to this pr, but the function for removing the gap file has 2 inputs that are not used:

- ctx context.Context
- cr *api.PerconaXtraDBCluster


We can drop them probably from the function.